week-2/H-array-literals/exercise.js

```diff
@@ -4,8 +4,8 @@
   Declare some variables assigned to arrays of values
 */
 
-var numbers = []; // add numbers from 1 to 10 into this array
-var mentors; // Create an array with the names of 4 mentors
+var numbers = [1, 6, 5, 5, 4]; // add numbers from 1 to 10 into this array
```

Comment:
I'm guessing you missed it at the bottom, but I don't think you're going to get the expected output with these numbers. Sometimes it's easiest to start with the expected results at the bottom to figure out where you're trying to get to!